topk-py/src/client/async/collection.rs

```diff
@@ -0,0 +1,139 @@
+use crate::data::value::{NativeValue, Value};
+use crate::error::RustError;
+use crate::query::{ConsistencyLevel, Query};
+use pyo3::prelude::*;
+use pyo3_async_runtimes::tokio::future_into_py;
+use std::{collections::HashMap, sync::Arc};
+
+#[pyclass]
+pub struct AsyncCollectionClient {
+    client: Arc<topk_rs::Client>,
+    collection: String,
+}
+
+impl AsyncCollectionClient {
+    pub fn new(client: Arc<topk_rs::Client>, collection: String) -> Self {
+        Self { client, collection }
+    }
+}
+
+#[pymethods]
+impl AsyncCollectionClient {
+    #[pyo3(signature = (ids, fields=None, lsn=None, consistency=None))]
+    pub fn get(
+        &self,
+        py: Python<'_>,
+        ids: Vec<String>,
+        fields: Option<Vec<String>>,
+        lsn: Option<String>,
+        consistency: Option<ConsistencyLevel>,
+    ) -> PyResult<PyObject> {
+        let client = self.client.clone();
+        let collection = self.collection.clone();
```

Comment:
nit: could we change `collection` from `String` to `Arc<String>` so cloning here is cheaper?

Reply:
updated in https://github.com/topk-io/topk/pull/233/commits/71d9a14ae1ddea5990f22d0a4a32745e511bdd33